build/all/Dockerfile

```diff
@@ -55,6 +55,7 @@ ARG USER_GID=$USER_UID
 RUN groupadd --gid $USER_GID $USERNAME && \
   useradd --uid $USER_UID --gid $USER_GID -m $USERNAME
 USER nonroot:nonroot
+RUN apt-get update && apt-get install -y bash git
```

Comment:
This puts bash in hydration-controller-with-shell. Is that desirable?

Reply:
It wasn't there before but I feel like it should be fine. The debian base image already comes with a shell, this just adds bash as another shell. The benefit is that we only need to perform the install in one layer

Reply:
If it's a big deal, we could cache everything except for the bash install and just `apt-get install bash` in the layer for the `nomos` image

Reply:
any idea what the size diff is with or without bash?

Reply:
As in the diff in the size of the resulting image?

Reply:
right. like, is it going to significantly increase costs for our users in download & storage? is it like 1MiB, or 100MiB?

Reply:
Based on local A/B testing:

Size without bash: 325279874 Bytes
Size with bash: 327555914 Bytes

Difference: 2276040 Bytes (Approximately 2 MB)

Diving into the container itself, it looks like the bash binary is a little over 1 MB:
```
-rwxr-xr-x 1 root root   1265648 Apr 23  2023  bash
```

Reply:
Thanks for checking. That seems fine.